modules/reference/pages/cluster-properties.adoc

```diff
@@ -1,1796 +1,5372 @@
-`kafka_throughput_limit_node_in_bps` or `kafka_throughput_limit_node_out_bps` can result in lesser throughput than kafka_quota_balancer_min_shard_throughput_bps.
+This property can be overridden on a per-topic basis by setting `retention.local.target.ms` in each topic enabled for Tiered Storage. See xref:manage:cluster-maintenance/disk-utilization.adoc#configure-message-retention[Configure message retention].
 
-Both <<kafka_quota_balancer_min_shard_throughput_ratio,kafka_quota_balancer_min_shard_throughput_ratio>> and kafka_quota_balancer_min_shard_throughput_bps can be specified at the same time. In this case, the balancer will not decrease the effective shard quota below the largest bps value of each of these two properties.
+NOTE: Both `retention_local_target_bytes_default` and `retention_local_target_ms_default` can be set. The limit that is reached earlier is applied.
 
-If set to `0`, no minimum is enforced.
+*Units*: milliseconds
 
-*Units*: bytes per second
 
-*Default*: 256
 
-*Range*: [0, ...]
+*Related properties*:
 
-*Restart required*: no
+* <<retention_local_target_bytes_default,retention_local_target_bytes_default>>.
 
-*Related topics*:
+*Requires Restart:* No
 
-* xref:manage:cluster-maintenance/manage-throughput.adoc#node-wide-throughput-limits[Node-wide throughput limits]
+*Nullable:* No
+
+*Visibility:* `user`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
 
-*Supported versions*: Redpanda v23.1 or later
+*Default:* `24h`
 
 ---
 
-=== kafka_quota_balancer_window_ms
+=== retention_local_trim_interval
 
-Time window used to average the current throughput measurement for the quota balancer.
+The period that disk usage is checked for disk pressure, and data is optionally trimmed to meet the target.
 
-*Units*: milliseconds
+*Units*: seconds.
 
-*Default*: 5000
+*Requires Restart:* No
 
-*Range*: [1, ...]
+*Nullable:* No
 
-*Restart required*: no
+*Visibility:* `tunable`
 
-*Related topics*:
+*Type:* integer
 
-* xref:manage:cluster-maintenance/manage-throughput.adoc#node-wide-throughput-limits[Node-wide throughput limits]
+*Accepted values:* [`-17592186044416`, `17592186044415`]
 
-*Supported versions*: Redpanda v23.1 or later
+*Default:* `30s`
 
 ---
 
-=== kafka_rpc_server_tcp_recv_buf
+=== retention_local_trim_overage_coeff
 
-Size of the Kafka server TCP receive buffer. If `null`, the property is disabled.
+The space management control loop will reclaim the overage multiplied by this this coefficient in order to compensate for data that is written during the idle period between control loop invocations.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* number
+
+*Default:* `2.0`
+
+---
+
+=== rm_sync_timeout_ms
+
+Resource manager's synchronization timeout. Maximum time for this node to wait for internal state machine to catch up with all events written by previous leaders before rejecting a request.
+
+*Units*: milliseconds.
```

Comment:
```suggestion
*Units*: milliseconds
```

Reply:
@Deflaimun Can you perform a global removal of all end punctuation following Units? thx